client/state/logstash/actions.ts

```diff
@@ -26,9 +25,6 @@ interface LogToLogstashParams {
  * @param params wpcom log2logstash params.
  * @returns      Action object
```

Comment:
Oh, and let's fix the docs in the meantime. This no longer returns an action object.